CHANGELOG.rst

```diff
@@ -4,7 +4,8 @@ Changelog
 master
 ------
 
-- (`#218 <https://github.com/openscm/scmdata/pull/218>`_) Replaced internal calls to ``RunGroupby.map`` with ``RunGroupby.apply``
+- (`#221 <https://github.com/openscm/scmdata/pull/221>`_) Add option to :func:`scmdata.ScmRun.interpolate` to allow for interpolation which ignores leap-years. This also fixes a bug where :func:`scmdata.ScmRun.interpolate` converts integer values into unix time. This functionality isn't consistent with the behaviour of the TimePoints class where integers are converted into years.
```

Comment:
Which test checks this bug doesn't return? Can we mark the relevant test with a comment e.g. `# check int not converted to unix time (gh 221)`?